### ISSUE TITLE
Option to get currently loaded Grafana theme

### ISSUE THREAD

**Issue by miq28** [ACE-IoT-Solutions/ace-svg-react#33]
Is there a way to get information about the currently loaded Grafana theme using this plugin?

I have looked at 'option' object but it seems this option is not available there.

I want to dynamically change the color of the line / path element based on currently loaded Grafana theme.

**Comment by nicfv** [comment]
I think there might be a way to achieve this! I'll look into it and keep you posted.

**Comment by nicfv** [comment]
@LarsStegman yes, sorry for the delays. I've figured out how to achieve this and I'll be implementing it in the next minor version of this plugin (`0.2.0`). I'll keep you all posted through development.
